e2e/configs.ts

```diff
@@ -42,6 +42,10 @@ export const ADMIN_FULL_NAME = 'Administrator';
 
 export const E2E_ROOT_PATH = __dirname;
 
+// Dates
+export const DATE_FORMAT = 'MMM DD, YYYY';
+export const DATE_TIME_FORMAT = 'MMM D, YYYY, h:mm';
```

Comment:
let's change the hour format to H:mm and also in the app.config - to have a 24h time format and avoid confusions

Reply:
or the correct format supported by moment -here

Reply:
this format is used in metadata tests that do not support app.config yet

Reply:
ok, understood